integreat_cms/cms/forms/contacts/contact_form.py

```diff
@@ -0,0 +1,115 @@
+from __future__ import annotations
+
+import logging
+from typing import TYPE_CHECKING
+
+from django import forms
+from django.utils.translation import gettext_lazy as _
+
+from ...models import Contact, POI
+from ..custom_model_form import CustomModelForm
+
+if TYPE_CHECKING:
+    from typing import Any
+
+logger = logging.getLogger(__name__)
+
+
+class ContactForm(CustomModelForm):
+    """
+    Form for creating and modifying contact objects
+    """
+
+    class Meta:
+        """
+        This class contains additional meta configuration of the form class, see the :class:`django.forms.ModelForm`
+        for more information.
+        """
+
+        #: The model of this :class:`django.forms.ModelForm`
+        model = Contact
+        #: The fields of the model which should be handled by this form
+        fields = [
+            "title",
+            "name",
+            "location",
+            "email",
+            "email_from_location",
+            "phone_number",
+            "phone_number_from_location",
+            "website",
+            "website_from_location",
+        ]
+
+    def __init__(self, **kwargs: Any) -> None:
+        r"""
+        Initialize contact form
+
+        :param \**kwargs: The supplied keyword arguments
+        """
+        super().__init__(**kwargs)
+        self.fields["email"].required = False
+        self.fields["phone_number"].required = False
+        self.fields["website"].required = False
+
+    def clean(self) -> dict[str, Any]:
+        """
+        Validate the fields
+
+        :return: The cleaned form data
+        """
+        cleaned_data = super().clean()
+
+        related_location = POI.objects.filter(id=self.data["location"]).first()
```

Comment:
```suggestion
        related_location = self.instance.region.pois.filter(id=self.data["location"]).first()
```

I think we should also filter by region, just to make sure that no one can create a contact that references a poi from another region. That would require also passing the additional instance attributes in the post method of the view.